doc/ccc.txt

```diff
@@ -312,6 +312,47 @@ Default: {
 	format.
 
 
+                                                          *ccc-option-recognise*
+recognize ~
+table
+	These are settings for recognize color format.
+
+
+                                                    *ccc-option-recognise-input*
```

Comment:
minor typo: you are using British English "recognise" on the right side (`*ccc-option-recognise*`), while the option on the left is spelled in American English "recognize".